app/controllers/lists_controller.rb

```diff
@@ -0,0 +1,56 @@
+class ListsController < ApplicationController
+    before_action :set_board, only: [:new, :create, :edit, :update, :destroy]
+    before_action :set_list, only: [:edit, :update, :destroy] 
+
+    def new
+        @list = @board.lists.new
+    end
+
+    def create
+        @list = @board.lists.create(create_params)
+        if @list.save
+            redirect_to board_path(@board), notice: 'List created'
+        else
+            render :new
+        end
+    end
+
+    def edit
+    end
+
+    def update
+          if @list.update_attributes
+            redirect_to boards_path(@board), notice: 'List name changed!'
+          else
+            render :edit
+          end
+    end
+
+    def destroy
+        if @list.destroy!
```

Comment:
to rzuca exception ! zeby wyłapac exception to trzeba zrobic rescue 

poczytaj o roznicy destroy vs destroy!

Reply:
?